crates/bevy_core_pipeline/src/skybox/mod.rs

```diff
@@ -169,7 +172,7 @@ impl SpecializedRenderPipeline for SkyboxPipeline {
                     } else {
                         TextureFormat::bevy_default()
                     },
-                    blend: Some(BlendState::REPLACE),
+                    blend: None,
```

Comment:
I think we need to comment why these are `None` rather than `REPLACE` to make sure they don't get changed back and silently break something. But maybe the breakage isn't so silent and we have a CI example that would catch it straight away?